indico/MaKaC/webinterface/common/abstractFilters.py

```diff
@@ -135,7 +135,7 @@ def satisfies( self, abstract ):
         elif not abstract.getContribType():
             return self._showNoValue
         else:
-            return abstract.getContribType().getId() in self._values
+            return abstract.getContribType().getId() in [value.getId() for value in self._values]
```

Comment:
Right now you create a list (n iterations), then need another n iterations to check whether the id is in there. It doesn't matter here because there are not many elements but it doesn't const anything to use a set where appropriate